ik_solver_msgs/Configuration solution
string[] colliding_obj_first
string[] colliding_obj_second
bool out_of_bound
float64 feasibility # 0 collision, 1 fully feasible
float64 distance    # distance w.r.t. obstacles
time stamp
